### PR TITLE
Stage 1.1 - Markup- Api- Backend- Links in ReadMe

### PR DESCRIPTION
**Pull request by iameddysmith**
While creating the ui, I needed to link to my 3rd party api as there was over 1k cards that would have been generated and the sample data of 3 in constants was not enough to efficiently style the page and work through scenarios where the large data load would have kept the styling (i.e, 7 across, 50 min limit in CharacterSection). I want to be able to create users which can store cards to their own collection so integrated the 3rd party API in the back end (link in my readme).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ This is the frontend for the **MyPokedex** project, a React-based app that conne
 
 For the backend API, check out the [MyPokedex-Backend](https://github.com/iameddysmith/MyPokedex-Backend).
```
Comment (gennady-bars) [inline README.md:7]: You `backend` repo contains `node_modules`, which is too huge for downloading. Please, remove it from `github` 
image

**Review by gennady-bars — COMMENTED** [review]
(**Please, expand the general comment ↓**)

Dear Eddy, 

unfortunately I have to reject the project without review:




* In **Files changed** there must be the code that should be reviewed. So, all changes from the last sprint should be there. But now you have only 4 files with minor changes there. I'm not able to leave comments in your code. 
* You `backend` repo contains `node_modules`, which is too huge for downloading. Please, remove it from `github` 
* You **deployed app** should work with your api. You shouldn't send requests to `localhost` there

File: README.md (hunk 1 above)
Comment (iameddysmith) [reply README.md:7]: I have took time off to build this project and have been working tirelessly on this project. I cleared out the node_modules from the back end. I pushed all my current updates to my back end as well as this pull request.

**Review by gennady-bars — COMMENTED** [review]
(**Please, expand the general comment ↓**)

Dear Eddy, 

unfortunately I have to reject the project without review:


* Adding comments to the files doesn't show the **whole code** of the files when I can leave my review messages for you https://snipboard.io/49Qu5Y.jpg . Kevin will help you soon how to revert the commits
* In **Files changed** there must be the code that should be reviewed. So, all changes from the last sprint should be there. I'm not able to leave comments in your code.

**Pull request merged**